it/v2/connectors/PersistenceConnectorSpec.scala

```diff
@@ -2130,6 +2133,111 @@ class PersistenceConnectorSpec
     }
   }
 
+  "GET /traders/:EORI/movements/:movementType/:movementId/messages/:messageId/body" - {
+
+    def targetUrl(eori: EORINumber, movementType: MovementType, movementId: MovementId, messageId: MessageId) =
+      s"/transit-movements/traders/${eori.value}/movements/${movementType.urlFragment}/${movementId.value}/messages/${messageId.value}/body"
+
+    "On successful retrieval of an element, must return ACCEPTED" in forAll(
+      arbitrary[EORINumber],
+      arbitrary[MovementId],
+      arbitrary[MessageId],
+      arbitrary[MovementType]
+    ) {
+      (eori, movementId, messageId, movementType) =>
+        server.stubFor(
+          get(
+            urlEqualTo(targetUrl(eori, movementType, movementId, messageId))
+          )
+            .withHeader(HeaderNames.ACCEPT, equalTo(MimeTypes.XML))
```

Comment:
No need to add header with get request

Reply:
...no, this is absolutely needed -- we accept XML from persistence.